resolvers/patient.js

```diff
@@ -1,10 +1,65 @@
+/* eslint-disable prettier/prettier */
```

Comment:
i don't like this - if we do have to disable any linter rules, i'd like it to be only wrapped around the section that violates it instead of the entire file